### ISSUE TITLE
Exported BMP file size in header seems wrong

### ISSUE THREAD

**Issue by saikyun** [aseprite/aseprite#3640]
When saving as or exporting BMP in aseprite, I consistently get the wrong file size in the bmp's header.
For example, a completely white image has 1024 x 1024 pixels, and 32 bits per pixel, for a total of 4096kb. But the reported file size (bytes 2-6 in the header) says 3072kb.

Other I've tried is:

* 4x1, where 4 bytes is missing.
* 4x2, where 8 bytes is missing.

(I tried attaching the image but github doesn't support BMP.)

### Aseprite and System version
v1.2.40-arm64
macOS Monterey 12.5

**Comment by Gasparoken** [comment]
Thanks for this report @saikyun. I was able to reproduce it in the past and it was solved at https://github.com/aseprite/aseprite/issues/3534 via https://github.com/aseprite/aseprite/pull/3535/commits/59fff4ba9c8104d41e3b58c50e2dd3118c08987a

To force 24bpp + 'correct file size' a workaround is possible:

- Convert the bottom layer to Background (this layer have to be Visible).
- Save the sprite.
- If you want to continue working without the Background layer, do 'Undo'.

The fix will be included in the next release.
